src/components/views/spaces/QuickSettingsButton.tsx

```diff
@@ -56,7 +56,6 @@ const QuickSettingsButton = ({ isPanelCollapsed = false }) => {
             wrapperClassName="mx_QuickSettingsButton_ContextMenuWrapper"
             onFinished={closeMenu}
             managed={false}
-            focusLock={true}
```

Comment:
TODO: Test whether `focusLock` matters in this menu

Reply:
I guess we can restore this because it's only used on opt-in `ContextMenu`. Seems like these type of menus should be refactored to use normal `RovingTabIndex` mechanics

There are only two `focusLock={true}` `ContextMenu` in the app atm.